code/modules/reagents/reagent_containers/hypospray.dm

```diff
@@ -306,3 +306,11 @@
 	name = "combat hypospray"
 	desc = "A sleek black hypospray. Its needle has the ability to bypass armor."
 	reagents_to_add = FALSE
+
+/obj/item/reagent_containers/hypospray/autoinjector/sanasomnum
+	name = "sanasomnum autoinjector"
+	desc = "A special autoinjector loaded with outlawed biomechanical stem cells, inducing a regenerative coma so intense it can heal almost any injury - even broken bones, organ and brain damage, severed tendons, and arterial damage. Upon use one will fall immediately into a state of unconsciousness lasting roughly three to five minutes, arising completely healed. The only thing it cannot fix are organs that have been destroyed outright, or so much cumulative damage that death is all but certain."
+	volume = 20
+	amount_per_transfer_from_this = 20
```

Comment:
Do we really need 20 units at once of this chem?

Reply:
10 is consumed far too quickly and furthermore I think giving people multiple hypos is a touch too strong.